src/hooks/useIsMouseEnter.ts

```diff
@@ -1,7 +1,8 @@
 import { useCallback, useState, RefObject } from "react";
 import useEventListener from "./useEventListener";
 
-export default function useMergeRefs({ ref }: { ref: RefObject<HTMLElement> }) {
+// TODO can be replaced with useIsMouseOver
```

Comment:
no difference at all? maybe this hook (without reading it throughly, only by naming) checks for the `enter` event and `over` hook check for if the mouse is over something at any time

Reply:
Seems to me like no difference at all, yeah. Didn't replaced it, cause if replacing thought that might be needed to rename corresponding variables as well + also though if it might cause merge conflicts for Yulia